cli/tests/e2e/test_ci.py

```diff
@@ -258,7 +258,14 @@ def mock_autofix(request, mocker):
         },
         {  # Jenkins
             "JENKINS_URL": "some_url",
-            "GIT_URL": "https://github.com/org/repo.git",
+            "GIT_URL": "https://github.com/org/repo.git/",
```

Comment:
Wanted to test a trailing slash because I put a fix for it in the git url parser. (Test still passes 🙌 )